packages/aws-cdk-lib/aws-cloudwatch-actions/test/lambda.test.ts

```diff
@@ -151,20 +152,13 @@ def handler(event, context):
   Template.fromStack(stack).resourceCountIs('AWS::Lambda::Permission', 2);
 });
 
-test('throws when multiple alarms are created for the same lambda if feature flag is set to false', () => {
-  // GIVEN
+test('can use same lambda for same action multiple time', () => {
```

Comment:
Sorry, it accidentally became cryptic diff, but this PR just does the following:
1. adds a new test case for `can use same lambda for multiple alarms with the same id`
2. removes the existing test case for `throws when multiple alarms are created for the same lambda if feature flag is set to false`.

Reply:
I moved the added test far away from the removed test to make it more visible.